server/chat-plugins/suspect-tests.ts

```diff
@@ -219,28 +219,28 @@ export const commands: Chat.ChatCommands = {
 			return this.parse('/help suspects');
 		},
 
-		deletecoil: 'setcoil',
-		sc: 'setcoil',
-		dc: 'setcoil',
-		async setcoil(target, room, user, connection, cmd) {
+		deletebvalue: 'setbvalue',
+		sbv: 'setbvalue',
+		dbv: 'setbvalue',
+		async setbvalue(target, room, user, connection, cmd) {
 			checkPermissions(this);
 			if (!toID(target)) {
 				return this.parse(`/help ${cmd}`);
 			}
-			const [format, source] = this.splitOne(target);
-			const formatid = toID(format);
+			const [formatStr, source] = this.splitOne(target);
+			const format = Dex.formats.get(formatStr);
 			let bVal: number | undefined = parseFloat(source);
 			if (cmd.startsWith('d')) {
 				bVal = undefined;
 			} else if (!source || isNaN(bVal) || bVal < 1) {
 				return this.errorReply(`Specify a valid COIL B value.`);
 			}
-			if (!formatid || !Dex.formats.get(formatid).exists) {
+			if (!toID(formatStr) || !format.exists) {
```

Comment:
Technically the first check for the formatStr argument isn't required since Dex.formats.get sanitizes the string even if no format exists, but I left this in regardless